spec/features/course/course_user_profile_spec.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.feature 'Courses: CourseUser Profile' do
+  let(:instance) { create(:instance) }
+
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+    let(:course_student) { create(:course_student, :approved, course: course) }
+    let!(:achievement) { create(:course_user_achievement, course_user: course_student).achievement }
```

Comment:
if we don't eager load this, the achievement will not appear on the course student's profile